portfolio/src/main/webapp/css/style.css

```diff
@@ -71,17 +71,22 @@ body {
 	z-index: 1;
 }
 .nav-section-mobile:hover .dropdown-links {
-    background-color: black;
 	display: flex;
 	flex-direction: column;
 }
-@media (max-width: 700px) {
+@media (max-width: 1000px) {
 	.nav-section {
 		display: none !important;
 	}
 	.nav-section-mobile {
 		display: inline-block;
 	}
+    .greeting-container {
+        display: none !important;
```

Comment:
Just noticed that, so quick FYI, using !important is discouraged at Google. It is error prone and it typically means that there is a better way to build your class structure. You can give it a try (as an exercise) to get rid of these, but that's a #sidequest level.

Reply:
Ok got it! For some reason `display: none ` by itself didn't work. I will look for another work around for removing `!important`